### ISSUE TITLE
Error in example notebook at calc_R_from_A

### ISSUE THREAD

**Issue by cchwala** [pycomlink/pycomlink#88]
The following code in the current example notebook
```python
cml['R'] = pycml.processing.k_R_relation.calc_R_from_A(A=cml.A, L_km=cml.length, f_GHz=cml.frequency/1e9)
```
throws this error
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-15-85fd714b5dbf> in <module>
----> 1 cml['R'] = pycml.processing.k_R_relation.calc_R_from_A(A=cml.A, L_km=cml.length, f_GHz=cml.frequency/1e9)

~/code/pycomlink/pycomlink/processing/xarray_wrapper.py in inner(*args, **kwargs)
     58                             {loop_dim: loop_dim_id}
     59                         ).values
---> 60                     data_list.append(func(**kwargs))
     61                 return xr.DataArray(
     62                     data=np.stack(data_list),

~/code/pycomlink/pycomlink/processing/k_R_relation.py in calc_R_from_A(A, L_km, f_GHz, a, b, pol, R_min)
     54     # This ignores the numpy warning stemming from A >=0 where A contains NaNs
     55     with np.errstate(invalid="ignore"):
---> 56         R[~nan_index & (A >= 0)] = (A[~nan_index & (A >= 0)] / (a * L_km)) ** (1 / b)
     57         R[~nan_index & (R < R_min)] = 0
     58     return R

~/mambaforge/envs/pycomlink-dev/lib/python3.7/site-packages/xarray/core/arithmetic.py in __array_ufunc__(self, ufunc, method, *inputs, **kwargs)
     78             kwargs=kwargs,
     79             dask="allowed",
---> 80             keep_attrs=_get_keep_attrs(default=True),
     81         )
     82 

~/mambaforge/envs/pycomlink-dev/lib/python3.7/site-packages/xarray/core/computation.py in apply_ufunc(func, input_core_dims, output_core_dims, exclude_dims, vectorize, join, dataset_join, dataset_fill_value, keep_attrs, kwargs, dask, output_dtypes, output_sizes, meta, dask_gufunc_kwargs, *args)
   1132             join=join,
   1133             exclude_dims=exclude_dims,
-> 1134             keep_attrs=keep_attrs,
   1135         )
   1136     # feed Variables directly through apply_variable_ufunc

~/mambaforge/envs/pycomlink-dev/lib/python3.7/site-packages/xarray/core/computation.py in apply_dataarray_vfunc(func, signature, join, exclude_dims, keep_attrs, *args)
    269 
    270     data_vars = [getattr(a, "variable", a) for a in args]
--> 271     result_var = func(*data_vars)
    272 
    273     if signature.num_outputs > 1:

~/mambaforge/envs/pycomlink-dev/lib/python3.7/site-packages/xarray/core/computation.py in apply_variable_ufunc(func, signature, exclude_dims, dask, output_dtypes, vectorize, keep_attrs, dask_gufunc_kwargs, *args)
    742         if data.ndim != len(dims):
    743             raise ValueError(
--> 744                 "applied function returned data with unexpected "
    745                 f"number of dimensions. Received {data.ndim} dimension(s) but "
    746                 f"expected {len(dims)} dimensions with names: {dims!r}"

ValueError: applied function returned data with unexpected number of dimensions. Received 1 dimension(s) but expected 0 dimensions with names: ()
```

Unfortunately `calc_R_from_A` is covered by tests but, apparently, not tested good enough. The test coverage stems from the useage of `calc_R_from_A` in `waa_leijnse_2008_from_A_obs` in the WAA tests. 

But `waa_leijnse_2008_from_A_obs` has the following hack to assure that `L_km` is not a `xarray.Dataarray`
https://github.com/pycomlink/pycomlink/blob/020ac8731b78f03c0e5fe407854ce9f2b6d0e778/pycomlink/processing/wet_antenna.py#L175-L178
and hence avoid the problem that occurs in the example notebook.

As far as I remember the example notebook worked before adding the WAA Leijnse function, hence, the problem might have been introduced by adding `.values` here
https://github.com/pycomlink/pycomlink/blob/979d5595243a06278052023cc5f9aec09fbd8e12/pycomlink/processing/xarray_wrapper.py#L56-L59

This issue could be fixed for this specific case, or in a more general manner by refactoring the xarray wrapper as suggested in https://github.com/pycomlink/pycomlink/issues/86#issuecomment-867903145

**Comment by cchwala** [comment]
While I work on the solution for the xarray wrapper, the following is a simple fix:

Instead of `L_km=cml.length` just use `L_km=float(cml.length)` to make sure that `cml.length` is not an xarray.Dataarray (which, in this case does not have the dimension `channel_id` over which the wrapper wants to iterate)

**Comment by cchwala** [comment]
closed in #89
